### DISCUSSION TITLE
Welcome to TaiwanWeatherAQI Discussions!

### DISCUSSION THREAD

**Discussion by Proliantaholic** [Proliantaholic/TaiwanWeatherAQI#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kemonohito** [comment]
<a href="https://imgur.com/eTe2geR"><img src="https://i.imgur.com/eTe2geR.png" title="source: imgur.com" /></a>

不好意思 我的變成這樣 該如何解決呢 已試過重新安裝
Rainmeter版本 4.4.0 beta

**Reply by Proliantaholic** [reply]
請更新到TaiwanWeatherAQI v2.01
